packages/color-kit-block/src/ColorKitBlock.tsx

```diff
@@ -0,0 +1,92 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import 'tailwindcss/tailwind.css';
+import '@frontify/fondue-tokens/styles';
+
+import { FC, ReactElement } from 'react';
+import {
+    Badge,
+    Button,
+    ButtonStyle,
+    IconArrowCircleDown,
+    Stack,
+    Text,
+    Tooltip,
+    TooltipPosition,
+    useCopy,
+} from '@frontify/fondue';
+import { useBlockSettings } from '@frontify/app-bridge';
+
+import { useColorPalettes } from './useColorPalettes';
```

Comment:
As you now have the color palettes/colors react hook +- ready, you can link it with `pnpm link ../../../app-bridge` (`pnpm link <path to local app-bridge>`) and don't forget to build appbridge (`pnpm build`)